src/main/java/Duke.java

```diff
@@ -7,6 +7,8 @@
      *
      */
     public static final String GREETING = " Hello! I'm Duke! What can I do for you?";
+
+    public static final String INVALID = " Invalid command!";
     public static final String GOODBYE = " Bye. Hope to see you again soon!";
     public static final String LINE_DIVIDER = "____________________________________________________________";
     public static final int INT = 101;
```

Comment:
maybe can improve the naming for this `INT` to be more specific and clear?

Reply:
Thank you. I will change it.